awl/mslider.cpp

```diff
@@ -239,14 +241,14 @@ void MeterSlider::paintEvent(QPaintEvent* ev)
             h  = y1 + lrint(i * mh / range);
             s.setNum(i);
             if (i == 0) {
-                  p.drawText(QRect(0, h - 3, 15, 9), Qt::AlignRight, QString::fromStdString("dB"));
-                  p.drawLine(18, h + 1, 23, h + 1);
+                  p.drawText(QRect(0, h - 3, METER_LEFT_EDGE - 3, 9), Qt::AlignRight, QString::fromStdString("dB"));
+                  p.drawLine(METER_LEFT_EDGE - 1, h + 1, METER_LEFT_EDGE - 1, h + 1);
                   continue;
                   }
             else if (i == range)
                   h -= 2;
-            p.drawText(QRect(0, h - 3, 15, 9), Qt::AlignRight, QString::fromStdString("-") + s);
-            p.drawLine(18, h + 1, 23, h + 1);
+            p.drawText(QRect(0, h - 3, METER_LEFT_EDGE - 3, 9), Qt::AlignRight, QString::fromStdString("-") + s);
+            p.drawLine(METER_LEFT_EDGE - 1, h + 1, METER_LEFT_EDGE - 1, h + 1);
```

Comment:
note that even though the notches are still drawn with a drawLine, the notches appear as a 2x2 pixel square area just to the left of the meter's left edge:

![notch-shifted-left_](https://cloud.githubusercontent.com/assets/6502474/24146287/74eea504-0e0b-11e7-8dbd-ecadf596d7e0.png)

There is not any more horizontal space available for the notches.  I'm happy with how it looks now.